airflow/models.py

```diff
@@ -2355,10 +2355,17 @@ def __deepcopy__(self, memo):
                     setattr(result, k, copy.deepcopy(v, memo))
                 except Exception as e:
                     logging.exception('Failed deepcopy on member variable '
-                                      'key: {k} ,value: {v}'.format(k=k, v=v))
+                                      'key: {k}, key type: {tk}, value: {v}, value type: {tv}'.format(
+                                        k=k, tk=type(k), v=v, tv=type(v)))
                     raise e
             else:
-                setattr(result, k, copy.copy(v))
+                try:
```

Comment:
I wonder if we can just have one try...except block for it seems very similar. What do you think @youngyjd ?

Reply:
sure